linters/trivy/plugin.yaml

```diff
@@ -26,23 +26,40 @@ tools:
 lint:
   definitions:
     - name: trivy
-      files: [lockfile, docker, yaml, terraform]
       tools: [trivy]
       suggest_if: files_present
       known_good_version: 0.37.1
       commands:
-        - name: fs
+        - name: fs-vuln
+          files: [lockfile]
           output: sarif
-          run: trivy fs ${target} --format json --no-progress --cache-dir ${shared_cachedir}
+          run:
+            trivy fs ${target} --scanners vuln --format json --no-progress --cache-dir
+            ${shared_cachedir}
           success_codes: [0]
           read_output_from: stdout
           # Trivy does not support batching
           batch: false
           is_security: true
           parser:
             runtime: python
-            run: python3 ${plugin}/linters/trivy/trivy_fs_to_sarif.py
+            run: python3 ${plugin}/linters/trivy/trivy_fs_vuln_to_sarif.py
+        - name: fs-secret
+          files: [ALL]
```

Comment:
Before we end up turning this on for everyone, couple verification questions:
1. How does performance compare to something like trufflehog?
2. How does secret scanning effectiveness compare to something like trufflehog?
3. What's the relative noisiness for our monorepo?

If any of these answers are in the negative direction, we may want to consider disabling this subcommand by default.

Reply:
I think this should be disabled by default.

1. It's slower than trufflehog.
2. It's about the same, I think.
3. Almost none - it has the added benefit of default ignoring our many fake secrets in test files.

Despite these, it has a few benefits over trufflehog - you can define your own regexes to search for and you can define regexes to not alert on (e.g. if you only use AWS keys, you can only search for those, making this tool faster). I think having trufflehog enabled for everybody, and then trivy secret scanning available but disabled by default for users looking to customize their secret scanning further is the best way to go.